### PR TITLE
Add support for Bootstrap alert types

### PR DESCRIPTION
**Pull request by maco**
* All alerts are currently shoved into either red (error) or green (success)
* Add support for yellow (warning) and blue (info) alert types

Solution from this Stack Overflow: http://stackoverflow.com/questions/31094771/rails-bootstrap-haml-how-to-convert-this-code-to-display-flash-messages-to

---

In reviewing #933 I discovered we don't presently have a way for a flash message to display as Bootstrap's `.alert-info` class. This would add that.

### CODE REVIEW

**Comment by pozorvlak** [comment]
The code looks fine, but we shouldn't get into the habit of ignoring "coverage decreased" messages - could you add a couple of simple unit tests for the new function?

**Comment by maco** [comment]
Yeah, will do. It was just getting late.

**Comment by maco** [comment]
hm darn. i closed and reopened to trip travis, but it didn't do what i hoped

**Comment by pozorvlak** [comment]
Coverage is now slightly up on dev, but I'd still like to see some simple unit tests for this.

**Comment by maco** [comment]
I don't know why it's up. I only closed and opened to test Code Climate. I
was infrastructure-ing last night.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -22,7 +22,22 @@ def forex_link(price)
...
       link,
       target: "_blank"
-  end 
+  end
+
+  def build_alert_classes(alert_type)
+    classes = 'alert alert-dismissable '
+    case alert_type.to_sym
+      when :alert, :danger, :error, :validation_errors
+        classes += 'alert-danger'
```
Comment (CloCkWeRX) [inline app/helpers/application_helper.rb]: Might be neater to make classes an array and .join(" ") everything at the end

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -22,7 +22,22 @@ def forex_link(price)
...
+      when :warning, :todo
+        classes += 'alert-warning'
+      when :notice, :success
+        classes += 'alert-success'
+      else
+        classes += 'alert-info'
```
Comment (CloCkWeRX) [inline app/helpers/application_helper.rb]: Should we instead raise an unknown alert type error; and encourage people to explicitly choose that its a .info; or put it a smart default ```build_alert_classes(alert_type = :info)```

Comment (maco) [inline app/helpers/application_helper.rb]: Oh, I like that default thing. I really just grabbed the Stack Overflow
suggestion.

**Comment by homu-gs** [comment]
:umbrella: The latest upstream changes (presumably #980) made this pull request unmergeable. Please resolve the merge conflicts.

**Comment by maco** [comment]
Code Climate was green, then I added one more test, and now...it's cached? When you click through, it shows complete and no new issues.

**Pull request merged**